tools/scaffolding/templates/gateway/frontend/package.json

```diff
@@ -33,9 +33,6 @@
     "react-scripts": "^4.0.1"
   },
   "workspaces": [
-    "clutch/frontend/api",
-    "clutch/frontend/packages/*",
-    "clutch/frontend/workflows/*",
```

Comment:
i assume this was snuck in here from private?

Reply:
yeah